packages/elixir_umbrella/config/config.exs

```diff
@@ -11,5 +11,8 @@ config :engine, :outgoing_booking_exchange, "outgoing_booking_updates"
 config :engine, :incoming_booking_exchange, "incoming_booking_updates"
 config :engine, :outgoing_plan_exchange, "outgoing_plan_updates"
 
-config :logger, level: :warn
+config :engine, :init_from_storage, System.get_env("INIT_FROM_STORAGE") == "true"
```

Comment:
This should be added to `releases.exs` config since we want to add variables `runtime` and not compile time